policy/modules/contrib/fdo.fc

```diff
@@ -0,0 +1,21 @@
+/usr/bin/fdo-admin-tool		--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/bin/fdo-owner-tool		--	gen_context(system_u:object_r:fdo_exec_t,s0)
+
+/usr/libexec/fdo/fdo-admin-tool			--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/libexec/fdo/fdo-manufacturing-client	--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/libexec/fdo/fdo-owner-onboarding-server	--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/libexec/fdo/fdo-rendezvous-server		--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/libexec/fdo/fdo-client-linuxapp		--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/libexec/fdo/fdo-manufacturing-server	--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/libexec/fdo/fdo-owner-tool			--	gen_context(system_u:object_r:fdo_exec_t,s0)
+/usr/libexec/fdo/fdo-serviceinfo-api-server	--	gen_context(system_u:object_r:fdo_exec_t,s0)
+
+/usr/lib/systemd/system/fdo-aio\.service			--	gen_context(system_u:object_r:fdo_unit_file_t,s0)
```

Comment:
I would just change this to 

/usr/lib/systemd/system/fdo.*\.service	

And remove the ones below.